app/component_fixtures/expandable_table.yml

```diff
@@ -0,0 +1,35 @@
+columns:
```

Comment:
I wonder if `app` folder is the right folder to put this kind of files. Should `config/component_fixtures` be a better fit ? just wondering, whatever you decide is fine.

Reply:
yeah grant and i deliberated on this for quite a while! there is no single place that’s the obvious right choice for this. we also strongly considered putting it in `config`. thinking about it, it’s not really configuration data, it’s more “data” data. we thought about adding a directory just called `data` somewhere in the directory structure, but again there wasn’t a place that such a directory seemed to fit logically and unambiguously. definitely there are multiple different places one could put this. in the end i went with `app` over `config` because the data doesn’t quite fit the category of configuration, but i still don’t feel that it’s the most graceful and natural place for it. if you can think of what its natural home is, let me know!

Reply:
no big deal at all, we could change it later if we find a better place. I would say let's leave it here for now. 👍